src/main/java/net/openhft/chronicle/wire/TextWire.java

```diff
@@ -3143,7 +3144,7 @@ public Object typePrefixOrObject(Class tClass) {
                         if (Wires.GENERATE_TUPLES) {
                             return Wires.tupleFor(null, stringBuilder.toString());
                         }
-                        throw new NoClassDefFoundError("Unable to load " + stringBuilder + ", is a class alias missing.");
+                        throw new MissingClassOrClassAliasException("Unable to load " + stringBuilder + ", is a class alias missing.");
```

Comment:
I feel like whatever happens we should probably stop throwing NoClassDefFoundError

Reply:
Perhaps the correct behaviour is to log a warning and return null, a lot of other ValueIns do that.

Reply:
Might be a better option